src/components/spaces/organisms/organelle-wrapper.tsx

```diff
@@ -265,6 +271,13 @@ export class OrganelleWrapper extends BaseComponent<OrganelleWrapperProps, Organ
     });
   }
 
+  private forceDropper(e: any) {
+    // Hack to force Fabric canvases to inherit cursor styles, should configure in Organelle instead
```

Comment:
This is the same hack I used in `connected-bio` to keep the dropper from turning back into a pointer...let me know if you think there's a cleaner solution.